### ISSUE TITLE
Regarding graph loss

### ISSUE THREAD

**Issue by kyang-06** [stoneMo/OsGG-Net#1]
Thank you for your great work!
I have one question about the implementation.

I notice that the training loss value may go below 0, caused by :
https://github.com/stoneMo/OsGG-Net/blob/93451cb1de121e13266f28c36744b48a630cd6b7/network/losses.py#L54
the subtraction operation  is confusing and conflicts with the paper.

I wonder if it is a mistake. And have you ever checked whether this codebase can reproduce the experimental results? (clean code may have reproduction problem)
